apps/native/app/src/screens/health/health-overview.tsx

```diff
@@ -0,0 +1,429 @@
+import { Alert, Button, Heading, Input, InputRow, Typography } from '@ui'
+import React, { useCallback, useState } from 'react'
+import { FormattedMessage, useIntl } from 'react-intl'
+import {
+  Image,
+  RefreshControl,
+  SafeAreaView,
+  ScrollView,
+  TouchableOpacity,
+  View,
+} from 'react-native'
+import { NavigationFunctionComponent } from 'react-native-navigation'
+import styled, { useTheme } from 'styled-components'
+
+import {
+  useGetHealthCenterQuery,
+  useGetHealthInsuranceOverviewQuery,
+  useGetPaymentOverviewQuery,
+  useGetPaymentStatusQuery,
+} from '../../graphql/types/schema'
+import { createNavigationOptionHooks } from '../../hooks/create-navigation-option-hooks'
+import externalLinkIcon from '../../assets/icons/external-open.png'
+import { getConfig } from '../../config'
+import { useBrowser } from '../../lib/use-browser'
+import { useConnectivityIndicator } from '../../hooks/use-connectivity-indicator'
+
+const Host = styled(SafeAreaView)`
+  padding-horizontal: ${({ theme }) => theme.spacing[2]}px;
+  margin-bottom: ${({ theme }) => theme.spacing[4]}px;
+`
+
+const ButtonWrapper = styled(View)`
+  flex-direction: row;
+  margin-top: ${({ theme }) => theme.spacing[3]}px;
+  gap: ${({ theme }) => theme.spacing[2]}px;
+`
+
+const { getNavigationOptions, useNavigationOptions } =
+  createNavigationOptionHooks((theme, intl) => ({
+    topBar: {
+      title: {
+        text: intl.formatMessage({ id: 'health.overview.screenTitle' }),
+      },
+    },
+  }))
+
+export const HealthOverviewScreen: NavigationFunctionComponent = ({
+  componentId,
+}) => {
+  useNavigationOptions(componentId)
+  const intl = useIntl()
+  const theme = useTheme()
+  const { openBrowser } = useBrowser()
+  const origin = getConfig().apiUrl.replace(/api$/, '')
+  const [refetching, setRefetching] = useState(false)
+
+  const healthInsuranceRes = useGetHealthInsuranceOverviewQuery()
+  const healthCenterRes = useGetHealthCenterQuery()
+  const paymentStatusRes = useGetPaymentStatusQuery()
+  const paymentOverviewRes = useGetPaymentOverviewQuery({
+    variables: {
+      input: {
+        dateFrom: intl.formatDate(new Date()),
+        dateTo: intl.formatDate(new Date()),
```

Comment:
This endpoint wants to get dateFrom and dateTo parameters but the things I am fetching from it (debt and credit) are static (I think - waiting for final confirmation from Sjúkratryggingar) so it should not make any difference what I send.